OMPython/__init__.py

```diff
@@ -1773,6 +1773,19 @@ def getLinearMatrixValues(self,matrix):
             return tmpmatrix
         else:
             return np.zeros((0,0))
+        
+    def setCurrDir(mo, newPath):
```

Comment:
Can you change this to `def setCurrDir(self, newPath):` so its consistent with the rest of the code.